website/project/model.py

```diff
@@ -1641,6 +1676,7 @@ def use_as_template(self, auth, changes=None, top_level=True):
         )
 
         new.save(suppress_log=True)
+        new.add_contributor(contributor=auth.user, permissions=CREATOR_PERMISSIONS, log=False, save=False)
```

Comment:
It's moved down to below the first save so that the subscription can be created (something something `_primary_key`s needing to exist before `ForeignField`s are populated. See previous PR for a more in-depth explanation).